csharp/src/Google.Protobuf/Reflection/MessageDescriptor.cs

```diff
@@ -34,6 +34,10 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
+#if DOTNET35
+// Needed for ReadOnlyDictionary, which does not exist in .NET 3.5
+using Google.Protobuf.Collections;
```

Comment:
Ah, maybe I just ported it and never noticed that it wasn't needed before. That helps simplify things.